lib/hexpm_web/templates/dashboard/email/index.html.eex

```diff
@@ -85,5 +85,42 @@
         </li>
       </ul>
     </div>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">Email Options</div>
+      <div class="panel-body">
+        <p>
+          Hex.pm always sends account-critical emails such as verification, password resets, and security alerts.
+          Below are the optional emails that you can turn off.
+        </p>
+      </div>
+
+      <%= form_for :optional_emails, ~p"/dashboard/email/options", [method: :post], fn f -> %>
+        <ul class="list-group">
+          <%= for type <- @optional_email_types do %>
+            <li class="list-group-item">
+              <div class="checkbox">
+                <label>
+                  <%= checkbox f, type.id,
+                    value: "true",
+                    checked: Map.get(@optional_email_preferences, to_string(type.id)) %>
+                  <strong><%= type.title %></strong>
+                </label>
+              </div>
+              <p>
+                <%= if Map.get(@optional_email_preferences, to_string(type.id), true) do %>
+                  <%= type.description_enabled %>
+                <% else %>
+                  <%= type.description_disabled %>
```

Comment:
Having the description change when the checkbox status changes feels confusing, it becomes a double negative.

The text "You will not be notified when an organization you belong to adds you." with the checkbox unchecked communicates the opposite of what we want imo. I would suggest removing `description_disabled`.